File: cs2100/Lecture/COde/MileToKm.c
// MileToKm.c
// Converts distance in miles to kilometers.
#include <stdio.h>
#define KMS_PER_MILE 1.609  

int main(void) {
	float miles,   // input - distance in miles. 
		  kms;     // output - distance in kilometers

	/* Get the distance in miles */
	printf("Enter distance in miles: ");
	scanf("%f", &miles);

	// Convert the distance to kilometres
	kms = KMS_PER_MILE * miles;

	// Display the distance in kilometres
	printf("That equals %9.2f km.\n", kms);

	return 0;
}

File: cs2100/Lecture/COde/MileToKm.py
# Conversion constant
KMS_PER_MILE = 1.609

# Get the distance in miles
miles = float(input("Enter distance in miles: "))

# Convert the distance to kilometres
kms = KMS_PER_MILE * miles

# Display the distance in kilometres
print("That equals", kms, "km.")
